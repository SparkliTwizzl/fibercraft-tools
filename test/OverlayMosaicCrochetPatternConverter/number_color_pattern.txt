0	red	10sc
1	blue	5sc,5dc
2	green	10dc
3	yellow	4sc,6dc
4	purple	10sc
5	orange	3dc,7sc
6	pink	10dc
7	brown	6sc,4dc
8	black	10sc
9	white	2dc,8sc
