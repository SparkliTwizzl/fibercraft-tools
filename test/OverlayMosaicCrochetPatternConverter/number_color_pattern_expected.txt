    0. red        bs, [1] 10sc, bs
    1. blue       bs, [1] 5sc, [6] 5dc, bs
    2. green      bs, [1] 10dc, bs
    3. yellow     bs, [1] 4sc, [5] 6dc, bs
    4. purple     bs, [1] 10sc, bs
    5. orange     bs, [1] 3dc, [4] 7sc, bs
    6. pink       bs, [1] 10dc, bs
    7. brown      bs, [1] 6sc, [7] 4dc, bs
    8. black      bs, [1] 10sc, bs
    9. white      bs, [1] 2dc, [3] 8sc, bs
